sdk/digitaltwins/azure-digitaltwins-core/samples/notebooks/01 Patrons.ipynb

```diff
@@ -0,0 +1,603 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# The Patron\n",
+    "## An example of the Digital Twin\n",
+    "\n",
+    "This example just includes:\n",
+    "* Connecting\n",
+    "* Uploading models\n",
+    "* Creating twins\n",
+    "* Querying twins\n",
+    "\n",
+    "It's not a lot, but it will all tie together when we start running scenarios. \n",
+    "\n",
+    "[This is the SDK repo on Github](https://github.com/Azure/azure-sdk-for-python/tree/4559e19e2f3146a49f1eba1706bb798071f4a1f5/sdk/digitaltwins/azure-digitaltwins-core)\n",
+    "\n",
+    "[Here is the doc on the query language](https://docs.microsoft.com/en-us/azure/digital-twins/concepts-query-language)\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Connecting\n",
+    "\n",
+    "**note**: you will need to replace {`home-test-twin.api.wcus.digitaltwins.azure.net`} with the address of your ADT. "
```

Comment:
Is there anyway we can configure this to pull the URL from an environment variable? Or some other configurable source?

Reply:
I could, however that would require extra steps from the analyst. AAD protects access to the site so it isn't a security issue (at least not on my side). 

My goal is to create notebooks that an analyst can use to quickly get up to speed on the process. Outside of setting up a conda environment, I would want to minimize the end-user setup.

Reply:
As a compromise, I changed this to:
```python
# you will get this from the ADT resource at portal.azure.com
your_digital_twin_url = "home-test-twin.api.wcus.digitaltwins.azure.net"

azure_cli = AzureCliCredential()
service_client = DigitalTwinsClient(
    your_digital_twin_url, azure_cli)
service_client
```